interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -2312,15 +2328,26 @@ public function getPlanOfCare($pid, $encounter)
             }
         }
 
-        $planofcare = '';
-        $query  = "SELECT 'care_plan' AS source,fcp.code,fcp.codetext,fcp.description,fcp.date,IF(sct_descriptions.ConceptId,'SNOMED-CT',ct.`ct_key`) AS fcp_code_type , l.`notes` AS moodCode
+        // some installations of OpenEMR do not have the SNOMED codes installed.  Rather than failing on a left join because
+        // the table does not exist we will include the SNOMED code pieces only if we have the sct_descriptions table installed.
+        // TODO: is there a better way to find out if the SNOMED tables have been installed through a global setting instead of describing the tables?
+        $fcp_code_type = 'ct.`ct_key` AS fcp_code_type';
+        $sct_descriptions_join = '';
+        $care_plan_query_data = ['Plan_of_Care_Type',$pid,'care_plan',0,$pid];
+        if ($this->is_snomed_codes_installed($appTable)) {
+            $fcp_code_type = "IF(sct_descriptions.ConceptId,'SNOMED-CT',ct.`ct_key`) AS fcp_code_type";
+            $sct_descriptions_join = ' LEFT JOIN sct_descriptions ON sct_descriptions.ConceptId = fcp.`code` 
+            AND sct_descriptions.DescriptionStatus = ? AND sct_descriptions.DescriptionType = ?
+            LEFT JOIN sct_concepts ON sct_descriptions.ConceptId = sct_concepts.ConceptId ';
+            $care_plan_query_data = array_merge([0,1], $care_plan_query_data);
+        }
+
+        $query  = "SELECT 'care_plan' AS source,fcp.code,fcp.codetext,fcp.description,fcp.date," . $fcp_code_type . " , l.`notes` AS moodCode
```

Comment:
TODO for me. Look into how the new snomed rf2 tables will be used instead of old rf1 tables.